internal/scheduler/schedulerapp.go

```diff
@@ -138,7 +138,7 @@ func Run(config schedulerconfig.Configuration) error {
 		executorRepository,
 		legacyExecutorRepository,
 		allowedPcs,
-		config.Scheduling.MaximumJobsToSchedule,
+		config.MaxJobsLeasedPerCall,
```

Comment:
I not that we've got *Maximum*JobsToSchedule but *Max*JobsLeasedPerCall- we should probably be consistent.